.github/workflows/ci.yml

```
@@ -62,6 +63,7 @@ jobs:
         run: "composer lint"
 
       - name: "Run coding style"
+        if: ${{ matrix.php-version != '8.1' }} # php-cs-fixer does not run on 8.1 yet
```

```suggestion
      - name: "Run coding style"
```

Testing if this is also still required.